README.md

```diff
@@ -1,56 +1,138 @@
+  </a>
+  <!-- NPM Downloads -->
+  <a href="https://www.npmjs.com/package/@frctl/fractal" title="NPM monthly downloads">
+    <img src="https://img.shields.io/npm/dm/@frctl/fractal" alt="">
+  </a>
+  <!-- License -->
+  <a href="https://github.com/frctl/fractal/blob/master/LICENSE" title="MIT license">
+    <img alt="GitHub" src="https://img.shields.io/github/license/frctl/fractal">
+  </a>
+</div>
+
+<br />
+
+Fractal is a tool to help you **build** and **document** web component libraries and design systems.
+
+[Read the full Fractal documentation][docs]
 
-Fractal is a tool to help you **build** and **document** web component libraries and then **integrate** them into your projects.
+## Introduction
 
-[![Build Status](https://img.shields.io/travis/frctl/fractal/master.svg?style=flat-square)](https://travis-ci.org/frctl/fractal)
-[![Greenkeeper badge](https://img.shields.io/badge/greenkeeper-enabled-brightgreen.svg?style=flat-square)](https://greenkeeper.io/)
-[![NPM Version](https://img.shields.io/npm/v/@frctl/fractal.svg?style=flat-square)](https://www.npmjs.com/package/@frctl/fractal)
+Component (or pattern) libraries are a way of designing and building websites in a modular fashion, breaking up the UI into small, reusable chunks that can then later be assembled in a variety of ways to build anything from larger components right up to whole pages.
 
-**Read the Fractal documentation at http://fractal.build/guide.**
+Fractal helps you assemble, preview and document website component libraries, or even scale up to document entire design systems for your organisation.
 
-**Need help?** We have an active and helpful community on [Fractal's Discord server](https://discord.gg/vuRz4Yx) - join us there for support and tips.
+Check out the the [documentation][docs] for more information.
 
-## Introduction
+## Requirements
 
-Component (or pattern) libraries are a way of designing and building websites in a modular fashion, breaking up the UI into small, reusable chunks that can then later be assembled in a variety of ways to build anything from larger components right up to whole pages.
+You'll need a [supported LTS version](https://github.com/nodejs/Release) of Node. Fractal may work on unsupported versions, but there is no active support from Fractal and new features may not be backwards compatible with EOL versions of Node.
 
-Fractal helps you assemble, preview and document website component libraries, and then integrate them into your web sites, apps and build processes to create joined up, 'living' projects.
+## Getting started
 
-Fractal can be run from the command line or integrated into your project via its API.
+### Install into your project (recommended)
 
-Check out the the [Fractal documentation](http://fractal.build/guide) for more information.
+```shell
+npm install @frctl/fractal
+```
 
-## Demo
+Then create your `fractal.js` file in the project root, and configure using the [official documentation][docs].
```

Comment:
hm, i think we should mention that on windows using `fractal.js` may lead to some problems when trying to run via the global cli. see https://github.com/frctl/fractal/issues/118#issuecomment-255254117

Reply:
(this config file name is one of the things i'd like to change, if we're going to start making small breaking changes)

Reply:
Done :)